### PR TITLE
Installers

### LINKED ISSUE #100: Create installers
## What ?
- [ ] Windows (.exe)
- [ ] Mac (?)
- [ ] Linux
  - [ ] .deb (debian package)
  - [ ] .tag.gz for other distributions
## How ?

I'm sure we'll have to implement some custom rules/build scripts, but I guess [electron-builder](https://www.npmjs.com/package/electron-builder) will help us get this done.

### LINKED ISSUE #335: Provide AppImage for Linux
Would you be interested in providing an [AppImage](http://appimage.org/) for Linux? It would have, among others, these advantages:
- Works for most Linux distributions (including Ubuntu, Fedora, openSUSE, CentOS, elementaryOS, Linux Mint, and others)
- One app = one file = super simple for users: just download one AppImage file, [make it executable](http://discourse.appimage.org/t/how-to-make-an-appimage-executable/80), and run
- No unpacking or installation necessary
- No root needed
- No system libraries changed
- Works out of the box, no installation of runtimes needed
- Optional desktop integration with `appimaged`
- Optional binary delta updates, e.g., for continuous builds (only download the binary diff) using AppImageUpdate
- Can optionally GPG2-sign your AppImages (inside the file)
- Works on Live ISOs
- Can use the same AppImages when dual-booting multiple distributions

[Here is an overview](https://github.com/probonopd/AppImageKit/wiki/AppImages) of projects that are already distributing upstream-provided, official AppImages.

If you have questions, AppImage developers are on #AppImage on irc.freenode.net.
I'm happy to help if there is interest.

### PR DESCRIPTION
**Pull request by KeitIG**
Fix #100
Fix #335 

### Platforms

- [x] Test Linux builds (Ubuntu)
- [x] Test Windows builds
- [ ] Test macOS builds

### Others 

- ~Keep portable versions or not?~

### Bugs

- [x] Tray not showing anymore on Linux

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "type": "git",
     "url": "https://github.com/KeitIG/museeks"
   },
-  "author": "Pierre de la Martinière",
+  "author": "Pierre de la Martinière <pierre.de.la.martiniere@gmail.com>",
```
Comment (YurySolovyov) [inline package.json]: Can be split into multiple fields: https://docs.npmjs.com/files/package.json#people-fields-author-contributors

Comment (KeitIG) [inline package.json]: > Or you can shorten that all into a single string, and npm will parse it for you `"Barney Rubble <b@rubble.com> (http://barnyrubble.tumblr.com/)"`

I guess it's fine

**Comment by KeitIG** [comment]
What annoys me is to create mac packages, I need a mac.

**Comment by KeitIG** [comment]
K, let's go. Will find a Mac or sth. Will see if I can make external builds.

**Pull request merged**